source/agora/consensus/validation/Transaction.d

```diff
@@ -86,7 +86,8 @@ public string isInvalidReason (
             return "Transaction: Output(s) overflow or underflow";
 
         // disallow 0 amount
-        if (output.value == Amount(0))
+        // TODO: remove Coinbase check when fees are calculated in TxBuilder
+        if (output.type != OutputType.Coinbase && output.value == Amount(0))
```

Comment:
Temporary until fees are calculated in TxBuilder

Reply:
Is updated in https://github.com/bosagora/agora/pull/2370 which has to be merged after this one.